oops/oops.go

```diff
@@ -198,7 +198,14 @@ func SkipFrames(err error, numFrames int) error {
 func (e *oopsError) writeStackTrace(w io.Writer) {
 	var base error
 	for err := error(e); err != nil; err = Unwrap(err) {
-		base = err
+		if _, ok := err.(*oopsError); ok {
+			// We've found another oops error in the chain, our "base" is no longer valid.
+			base = nil
+		} else if base == nil {
+			// We've found a non-oops error, and this is the first non-oops error we've
+			// found in the current chain.
```

Comment:
Can you give a brief example here, to make it concrete. Eg, the sendto call.

Reply:
done! (below)